src/commands/music/play.ts

```diff
@@ -7,6 +7,11 @@ import queueCommand from "./queue";
 const playCommand: Command = {
     name: "play",
     aliases: ["p"],
+    additionalHelpInfo: [
+        "**Special Cases:**",
+        "`!play --playlist playlistLink` will play a youtube playlist",
+        "`!play` will play from the queue if not empty",
```

Comment:
Replace the "!" with the environment variable for the prefix.

At the top of the file you can `import "dotenv/config"`

Also reference the name of this command using `this.name`